### PR TITLE
[zh-cn] sync basic-stateful-set.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md

### CODE REVIEW

File: content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -1823,20 +1865,51 @@ above.
...
+
+<!--
+If you added two more Pods, Kubernetes would set up `app-3` and wait for that to become healthy before deploying
+`app-4`.
+
+Because this is the default setting, you've already practised using it.
+-->
+如果你再添加两个 Pod，Kubernetes 将设置 `app-3` 并等待其正常运行，然后再部署 `app-4`。
+
+因为这是默认设置，所以你已经在使用它。
```
Comment (sftim) [inline content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md:1887]: The nuance to capture here is “&hellip;and by implication, the tutorial will not make you run through the same steps once again”.

File: content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 2]
```diff
@@ -531,6 +542,20 @@ to Running and Ready.
...
 
+<!--
+If your application wants to find any healthy Pod in a StatefulSet,
+and therefore does not need to track each specific Pod,
+you could also connect to the IP address of a `type: ClusterIP` Service,
+backed by the Pods in that StatefulSet. You can use the same Service that
+tracks the StatefulSet (specified in the `serviceName` of the StatefulSet)
+or a separate Service that selects the right set of Pods.
+-->
+如果你的应用程序想要在 StatefulSet 中找到任意健康的 Pod，
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md:553]:
```suggestion
如果你的应用程序想要在 StatefulSet 中找到任一健康的 Pod，
```

File: content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 3]
```diff
@@ -715,6 +741,14 @@ This is accomplished by updating the `replicas` field. You can use either
 -->
 ### 扩容   {#scaling-up}
 
+<!--
+Scaling up means adding more replicas.
+Provided that your app is able to distribute work across the StatefulSet, the new
+larger set of Pods can perform more of that work.
+-->
+扩容意味着添加更多副本。
+如果你的应用程序能够跨 StatefulSet 分配工作，则新的更大的 Pod 集可以执行更多的工作。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md:750]:
```suggestion
如果你的应用程序能够在整个 StatefulSet 范围内分派工作，则新的更大的 Pod 集可以执行更多的工作。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**